source/features/hide-empty-split-diff-side.tsx

```diff
@@ -0,0 +1,31 @@
+import './hide-empty-split-diff-side.css';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	for (const diffTable of select.all('.js-diff-table')) {
+		for (const side of ['left', 'right']) {
+			if (!select.exists(`[data-split-side="${side}"]:is(.blob-code-addition, .blob-code-deletion)`, diffTable)) {
+				diffTable.classList.add('rgh-empty-split-' + side);
```

Comment:
Use attributes
```suggestion
				diffTable.setAttribute('data-rgh-always-unified-diff', side);
```

This way both can be selected with just `[data-rgh-always-unified-diff]